clientV2/assets/components/_modal.scss

```diff
@@ -0,0 +1,68 @@
+$primaryGray: #777777;
```

Comment:
I think there is no need for creating this $primaryGray colour here because it is already in the assets/theme/_color.scss.

What you have to do is to import from `assets/theme/_color.scss` and do `color: map-get($colors, grey);` where you need it.